daml-script/test/daml/ScriptTest.daml

```diff
@@ -458,10 +458,10 @@ testUserManagement = do
   u <- getUser "nonexistent"
   u === None
   users <- listUsers
-  users === [User "u1" None, User "u2" None, participantAdmin]
+  users === [participantAdmin, User "u1" None, User "u2" None]
```

Comment:
@cocreature fyi

Reply:
Thanks for the ping lgtm, I should probably add a `sort` there at some point.